### PR TITLE
feat: unread archived conversations indicator [WPB-4437]

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-4437" title="WPB-4437" target="_blank">StoryWPB-4437</a>  Add an indicator on the archived tab 
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Added unread indicator for archived conversation drawer item

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

| Before | After |
| ----------- | ------------ |
|  before |  after |

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1053](https://10.10.124.134/job/AR-build-pipeline/1053/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7693860)(7693860)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7693860)(7693860)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [1063](https://10.10.124.134/job/AR-build-pipeline/1063/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7700721)(7700721)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7700721)(7700721)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/WireApplication.kt [hunk 1]
```diff
@@ -112,7 +112,7 @@ class WireApplication : Application(), Configuration.Provider {
                     .detectDiskReads()
                     .detectDiskWrites()
                     .penaltyLog()
-                    .penaltyDeath()
+//                    .penaltyDeath()
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/WireApplication.kt]: Did this slip to this PR mistakenly? Not sure if we agreed to this in collective. Last information I got was that we should keep the penalty death on debug builds and fix whatever was complaining or causing the crashes.

**Review by gongracr — APPROVED** [review]
Looking good. Just a minor non blocking comment

**Comment by AndroidBob** [comment]
Build [1068](https://10.10.124.134/job/AR-build-pipeline/1068/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7701371)(7701371)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7701371)(7701371)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [1086](https://10.10.124.134/job/AR-build-pipeline/1086/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7726376)(7726376)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7726376)(7726376)-staging-compat.apk)

**Pull request merged**